authn-authz/binding.md

```diff
@@ -76,45 +76,131 @@ $ tanzu auth --kubeconfig /tmp/pinniped_kubeconfig.yaml binding add --user usern
+tanzu rbac binding add --user $user --role $role --namespace $namespace
 
-tanzu auth binding add --group $group --role $role --namespace $namespace
+tanzu rbac binding add --group $group --role $role --namespace $namespace
 ```
 
 For example:
 
 ```
-$ tanzu auth binding add --user username@vmware.com --role app-editor --namespace user-ns
+$ tanzu rbac binding add --user username@vmware.com --role app-editor --namespace user-ns
 ```
 
 ### <a id="get-list-users"></a> Get a list of users and groups from a role
 
 Get a list of users and groups from a role by running:
 
 ```
-tanzu auth binding get --role $role --namespace $namespace
+tanzu rbac binding get --role $role --namespace $namespace
 
-tanzu auth binding get --role $role --namespace $namespace
+tanzu rbac binding get --role $role --namespace $namespace
 ```
 
 For example:
 
 ```
-$ tanzu auth binding get --role app-editor --namespace user-ns
+$ tanzu rbac binding get --role app-editor --namespace user-ns
 ```
 
 ### <a id="binding-delete"></a> Remove the specified user or group from a role
 
 Remove a user or group from a role by running:
 
 ```
-tanzu auth binding delete --user $user --role $role --namespace $namespace
+tanzu rbac binding delete --user $user --role $role --namespace $namespace
 
-tanzu auth binding delete --group $group --role $role --namespace $namespace
+tanzu rbac binding delete --group $group --role $role --namespace $namespace
 ```
 
 For example:
 
 ```
-$ tanzu auth binding delete --user username@vmware.com --role app-editor --namespace user-ns
+$ tanzu rbac binding delete --user username@vmware.com --role app-editor --namespace user-ns
 ```
+
+### <a id="error-logs"></a> Error logs
+
+#### Permission Denied:
+
+The current user does not have permissions to create or modify rolebinding objects.
+Please use an administrator account when using the rbac cli.
+```
+Error: rolebindings.rbac.authorization.k8s.io "app-operator" is forbidden: User "<subject>" cannot get resource "rolebindings" in API group "rbac.authorization.k8s.io" in the namespace "namespace"
+Usage:
+tanzu rbac binding add [flags]
+Flags:
+-g, --group string User Group
+-h, --help help for add-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Already Bound Error:
+Adding a subject (user or group) to a role that already has the subject will produce the following error:
+```
+Error: User ‘test-user’ is already bound to 'app-operator' role
+Usage:
+tanzu rbac binding add [flags]
+Flags:
+-g, --group string User Group
+-h, --help help for add-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Could Not Find Error:
+Removing a subject (user or group) from a role that does not exist will produce the following error:
```

Comment:
Is this supposed to be removing a subject from a role that exists, but doesn't have the subject bound to it, or a role that doesn't exist at all?